ntc_templates/templates/juniper_junos_show_interfaces.textfsm

```diff
@@ -2,14 +2,17 @@ Value Required INTERFACE (\S+)
 Value LINK_STATUS (\w+)
 Value ADMIN_STATE (\S+)
 Value HARDWARE_TYPE (\S+)
+Value DESCRIPTION (\w+.*)
+Value DESTINATION (\S+)
+Value LOCAL (\S+)
 Value MTU (\d+|Unlimited)
 
 Start
-  ^\s+Logical\s+interface\s+${INTERFACE}
   ^Physical\s+interface:\s+${INTERFACE},\s+${ADMIN_STATE},\s+Physical\s+link\s+is\s+${LINK_STATUS}
+  ^.*Description:\s+${DESCRIPTION}
   ^.*ype:\s+${HARDWARE_TYPE},.*MTU:\s+${MTU}.* -> Record
-  ^.*MTU:\s+${MTU}.* -> Record
```

Comment:
Was this intended to be deleted? If so, what is the reason to remove it?

Reply:
yes, as it didn't appear to work in my tests against field gear it didn't work across multiple platforms.

Reply:
I think that was because of the Record. My changes would make the Record deterministic on each new interface, so that should probably stay in as `^.*MTU\s+${MTU}`